File: widgetssdk/src/main/java/com/glia/widgets/core/visitor/VisitorMediaUpdatesListener.java
package com.glia.widgets.core.visitor;

import com.glia.androidsdk.comms.VisitorMediaState;

public interface VisitorMediaUpdatesListener {
    void onNewVisitorMediaState(VisitorMediaState visitorMediaState);

    void onHoldChanged(boolean isOnHold);
}

File: widgetssdk/src/main/java/com/glia/widgets/core/visitor/VisitorMediaUpdatesListener.kt
package com.glia.widgets.core.visitor

import com.glia.androidsdk.comms.VisitorMediaState

interface VisitorMediaUpdatesListener {
    fun onNewVisitorMediaState(visitorMediaState: VisitorMediaState?)
    fun onHoldChanged(isOnHold: Boolean)
}
